scripts/GenerateJSONOutput.py

```diff
@@ -50,25 +57,12 @@
 
 args = parser.parse_args()
 
-
 # Print out the settings
-for setting in dir(args):
-    if not setting[0] == "_":
-        print(
-            ">>> ... Setting: {: >20} {: >40}".format(
-                setting, eval("args.%s" % setting)
-            )
-        )
+for arg in vars(args):
+    user_input = getattr(args, arg)
+    print(f">>> ... Setting: {arg: >20} {str(user_input): >40}")
```

Comment:
@histfitter/developers When the argument is a list is the desired behavior like how I have it now which would give (I think this was the previous format)

```console
$ python scripts/GenerateJSONOutput.py -i file_A.root file_B.root -f hypo_C1C1_%f_%f
>>> ... Setting:           inputFiles           ['file_A.root', 'file_B.root']
>>> ... Setting:               format                          hypo_C1C1_%f_%f
>>> ... Setting:       interpretation                                   m0:m12
>>> ... Setting:       addCoordinates                     {"m0":"x","m12":"y"}
>>> ... Setting:                  cut                                        1
>>> ... Setting:            noAddTabs                                    False
```

or is the alternative of having them one per line preferred?

```console
$ python scripts/GenerateJSONOutput.py -i file_A.root file_B.root -f hypo_C1C1_%f_%f
>>> ... Setting:           inputFiles                              file_A.root
>>> ... Setting:           inputFiles                              file_B.root
>>> ... Setting:               format                          hypo_C1C1_%f_%f
>>> ... Setting:       interpretation                                   m0:m12
>>> ... Setting:       addCoordinates                     {"m0":"x","m12":"y"}
>>> ... Setting:                  cut                                        1
>>> ... Setting:            noAddTabs                                        0
```